spec/lib/vcr/library_hooks/faraday_spec.rb

```diff
@@ -10,45 +10,23 @@
     end
 
     conn.builder.lock!
-    expect(conn.builder.handlers.last(2).map(&:klass)).to eq([
-      VCR::Middleware::Faraday,
-      Faraday::Adapter::NetHttp
-    ])
+    expect(conn.builder.handlers.last.klass).to eq(VCR::Middleware::Faraday)
   end
 
   it 'handles the case where no adapter is declared' do
     conn = Faraday.new
 
     conn.builder.lock!
-    expect(conn.builder.handlers.last(2).map(&:klass)).to eq([
-      VCR::Middleware::Faraday,
-      Faraday::Adapter::NetHttp
-    ])
+    expect(conn.builder.handlers.last.klass).to eq(VCR::Middleware::Faraday)
   end
 
   it 'does nothing if the VCR middleware has already been included' do
     conn = Faraday.new(:url => 'http://sushi.com') do |builder|
       builder.use VCR::Middleware::Faraday
-      builder.use Faraday::Response::Logger
-      builder.use Faraday::Adapter::NetHttp
-    end
-
-    conn.builder.lock!
-    expect(conn.builder.handlers.map(&:klass)).to eq([
-      VCR::Middleware::Faraday,
-      Faraday::Response::Logger,
-      Faraday::Adapter::NetHttp
-    ])
-  end
-
-  it 'prints a warning if the faraday connection stack contains a middleware after the HTTP adapter' do
-    conn = Faraday.new(:url => 'http://sushi.com') do |builder|
-      builder.use Faraday::Adapter::NetHttp
-      builder.use Faraday::Response::Logger
```

Comment:
✂️ Nice@!  Yeah, agreed.  This test is no longer valid with faraday 1.0.0 as the warning doesn't happen because the adapter is managed differently now.